packages/api/app/Models/ServiceOrder.js

```diff
@@ -18,6 +18,14 @@ class ServiceOrder extends Model {
 	serviceOrderReviews() {
 		return this.hasMany('App/Models/ServiceOrderReview');
 	}
+
+	static get computed() {
+		return ['type'];
+	}
+
+	getType() {
+		return 'S';
```

Comment:
`S` significa `service`?

```suggestion
		return 'service';
```

Reply:
Esse S é de service, eu coloquei assim pq é um requisito da issue.

Reply:
Pq é um requisito?

Reply:
Acho que tanto faz, @nichollasrennah colocou lá